streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -60,8 +62,10 @@
 
     private boolean commitRequested = false;
     private boolean commitOffsetNeeded = false;
-
     private boolean requiresPoll = true;
+    private final Time time;
+    private final StreamsMetrics metrics;
+    private Sensor nodeCommitTimeSensor;
```

Comment:
I think this is not a per-node sensor, but rather a per-task sensor right? Shall we create a `TaskMetrics` class accordingly for this layer?